tests/ui/nll/issue-24535-allow-mutable-borrow-in-match-guard.rs

```diff
@@ -5,7 +5,6 @@
 // See further discussion on rust-lang/rust#24535,
 // rust-lang/rfcs#1006, and rust-lang/rfcs#107
 
-#![feature(if_let_guard)]
 
```

Comment:
There's a bunch of double newlines when you remove this feature line. If removing the feature line results in no features being used, then there should be no double newlines.